packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -383,7 +383,7 @@ function useAutoFocus(scopeRef: RefObject<HTMLElement[]>, autoFocus: boolean) {
       }
     }
     autoFocusRef.current = false;
-  }, []);
+  }, [scopeRef]);
```

Comment:
this change seems unnecessary, what is it for?
what test am i missing? I've added one, for the case you outlined, but it seems like maybe you're covering some others?

Reply:
> this change seems unnecessary, what is it for?
Fixed an eslint warning for missing dependency on useEffect.
> what test am i missing? I've added one, for the case you outlined, but it seems like maybe you're covering some others?
Is it possible to test the use case for when the focusedKey in the TableView is scrolled out of view?

Reply:
> Is it possible to test the use case for when the focusedKey in the TableView is scrolled out of view?

should be, you can find examples of how to scroll TableView in tests here https://github.com/adobe/react-spectrum/blob/main/packages/%40react-spectrum/table/test/Table.test.js#L1478

Reply:
ok, if it's just fixing eslint, i'm fine with it